electron/main.ts

```diff
@@ -27,7 +27,7 @@ function createWindow () {
     mainWindow = null;
   });
 
-  mainWindow.webContents.on('new-window', (event: Event, url: string) => {
+  mainWindow.webContents.on('new-window', (event, url) => {
```

Comment:
Been looking at this for a little while, and I have a pretty good understand of types and type-safety in TS. 

From looking at Electron documentation, I'm pretty sure that `event` and `url` are automatically defined as type `NewWindowWebContentsEvent` and `string` respectively after being returned from the `'new-window'` event. When I hover my mouse over `event` and `url` in VSCode (in the new line), it shows that it has the correct type. So, declaring the type seems redundant. 

I would have loved to declare the type and keep the line as it was, but when I do I keep getting the following error:
`SyntaxError: Unexpected token ':'`
The same error shows when I tried to declare the types elsewhere in the file. 

I did find, however, that this `'new-window'` event within the `webContents` class is actually deprecated. The new method seems fairly complicated, though, and I don't know if it would work within the current setup.

Regardless, I am stuck.

Reply:
Overall, I think you made a good point and we can revisit the type issue in the future is absolutely needed.